packages/calcite-components/src/tests/commonTests/accessible.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jest/no-export -- Util functions are now imported to be used as `it` blocks within `describe` instead of assertions within `it` blocks. */
```

Comment:
This, and similar, disabling lines should be removed in favor of the `commonTests` ESLint config file.